backend/src/realtime/realtime-note/realtime-note.service.spec.ts

```diff
@@ -89,15 +106,78 @@ describe('RealtimeNoteService', () => {
     setIntervalSpy = jest.spyOn(global, 'setInterval');
     clearIntervalSpy = jest.spyOn(global, 'clearInterval');
 
+    clientWithReadWrite = new MockConnectionBuilder(realtimeNote)
+      .withAcceptingRealtimeUserStatus()
+      .withLoggedInUser(readWriteUsername)
+      .build();
+
+    clientWithRead = new MockConnectionBuilder(realtimeNote)
+      .withDecliningRealtimeUserStatus()
+      .withLoggedInUser(onlyReadUsername)
+      .build();
+
+    clientWithoutReadWrite = new MockConnectionBuilder(realtimeNote)
+      .withDecliningRealtimeUserStatus()
+      .withGuestUser(noAccessUsername)
+      .build();
+
+    clientWithoutReadWrite.getTransporter().sendReady();
+    clientWithReadWrite.getTransporter().sendReady();
```

Comment:
Actually.. there is no communication happening here. So no need for the ready messages 🤔

Reply:
removed sendReady method call